sdk_generator.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
```

Comment:
I'm pretty sure I had Fx-iOS set up to use the sdk_generator.sh that was checked out as part of the carthage release, so using the one in the Carthage/Build/iOS/Glean dir rather than a static one that needed to be kept up to date.

Reply:
I can check whether carthage does the submodule checkout, which would have the file again

Reply:
It seems to do a full checkout!